packages/input-edit/src/modules/inputEditTable/reducer.js

```diff
@@ -0,0 +1,39 @@
+import {reducer as reducerUtil} from 'tocco-util'
+
+import * as actions from './actions'
+
+const initialState = {
+  data: [],
+  inputEditForm: [],
+  inputDataForm: {},
+  sorting: {}
+}
+
+const setValue = (state, action) => {
+  const {inputDataKey, node, value} = action.payload
+  return {
+    ...state,
+    data: state.data.reduce((val, acc) => {
```

Comment:
ok sorry my fault, the first parameter is the accumulator (acc) and the second is the currentValue (value). you sould probably rename them. sorry :)